src/hooks/useAuth.jsx

```diff
@@ -0,0 +1,7 @@
+import { useContext } from "react";
+import { AuthContext } from "../contexts/AuthContext";
+
+export default function useAuthContext() {
+  const auth = useContext(AuthContext);
+  return auth;
```

Comment:
```suggestion
  return useContext(AuthContext);
```

で良さそうかな？と思いました！

利用側は const { isLoggedIn, setIsLoggedIn } = useAuthContext(); のように使えるはず！